src/main/java/team/creative/creativecore/common/gui/controls/collection/GuiStackSelector.java

```diff
@@ -259,15 +258,17 @@ public static class GuiBlockSelector extends SearchSelector {
         
         @Override
         public boolean allow(ItemStack stack) {
-            if (super.allow(stack))
-                return Block.byItem(stack.getItem()) != null && !(Block.byItem(stack.getItem()) instanceof AirBlock);
+            if (super.allow(stack)) {
+                Block.byItem(stack.getItem());
+                return !(Block.byItem(stack.getItem()) instanceof AirBlock);
```

Comment:
_my brain at some point collapses and adds another Block.byItem for no reason_
i remove the nullcheck because result was never null, `instanceof` checks also checks if it wasn't null. and null is not an instance of null

the extra Block.byItem line removed too
![image](https://github.com/CreativeMD/CreativeCore/assets/29090346/71d46042-7350-47d9-a89c-a386f15cd39c)